guide/english/html/index.md

```diff
@@ -42,8 +42,9 @@ title: The element specifies a title for the document
 body: The element contains the visible page content
 
 h1: The element defines a large heading
-
+There are 6 Heading Tag in HTML depending on their inportance.
```

Comment:
This is not the place for this description, it is better to write in a separate file in more detail about heading tags.